### PR TITLE
📦 update types

### PR DESCRIPTION
**Pull request by jerembdn**

### CODE REVIEW

File: organization/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  updatedAt: Date;
+  createdAt: Date;
+};
+
+export type OrganizationSocial = {
+  type: OrganizationSocialType;
+  url: string;
+};
+
+export type OrganizationSocialType = "facebook" | "twitter" | "instagram" | "linkedin" | "youtube" | "website" | string;
```
Comment (AntoineKM) [inline organization/index.d.ts:23]: Wouldn't it be better in Enum form?

File: tokens/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+export type UserToken = {
+  id: string;
+  type: UserTokenType;
+  value: string;
+  createdAt: Date;
+  expiresAt: Date;
+}
+
+export type UserTokenType = "recovery" | "email" | "phone";
```
Comment (AntoineKM) [inline tokens/index.d.ts:9]: Wouldn't it be better in Enum form?

File: user/index.d.ts [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  firstName: string;
+  lastName: string;
+  userName: string;
+  gender: UserIdentityGender;
+  profilePictureUrl?: string;
+  birthDate: Date;
+  // idValid: boolean;
+};
+
+export type UserIdentityGender = "male" | "female" | "non-binary" | string;
```
Comment (AntoineKM) [inline user/index.d.ts:30]: Wouldn't it be better in Enum form?

File: index.d.ts [hunk 4]
```diff
@@ -152,3 +6,9 @@ export type Address = {
   city: string;
   country: string; // was missing
 };
+
+// Currency
+export type Currency = "eur" | "usd";
```
Comment (AntoineKM) [inline index.d.ts:11]: CurrencyType, enum ?

File: index.d.ts [hunk 5]
```diff
@@ -152,3 +6,9 @@ export type Address = {
...
+
+// I18n
+export type Language = "fr" | "en";
```
Comment (AntoineKM) [inline index.d.ts:14]: Language Type, enum ?

File: tokens/index.d.ts (hunk 2 above)
Comment (AntoineKM) [inline tokens/index.d.ts:9]: add ``organization-invite`` token type ?

**Review by AntoineKM — CHANGES_REQUESTED** [review]

File: organization/index.d.ts (hunk 1 above)
Comment (jerembdn) [reply organization/index.d.ts:23]: In the database and graphql for optimization, theses types will be oriented in this idea of conception.
Let's keep this use before we change it later

File: tokens/index.d.ts (hunk 2 above)
Comment (jerembdn) [reply tokens/index.d.ts:9]: mhh same as socialtype, what's better ?

File: user/index.d.ts (hunk 3 above)
Comment (jerembdn) [reply user/index.d.ts:30]: Same, let me know your opinion

File: tokens/index.d.ts (hunk 2 above)
Comment (jerembdn) [reply tokens/index.d.ts:9]: It's already in the next update but in another branch (and authentication token type added too)

**Review by AntoineKM — APPROVED** [review]

**Pull request merged**